### ISSUE TITLE
Releases unavailable

### ISSUE THREAD

**Issue by jufemaiz** [ariga/atlas#3213]
Hi team, seems like the releases website has had a number of releases deleted.

v0.19.0 and prior appear to have been removed. Can you please confirm?

**Comment by a8m** [comment]
Hey @jufemaiz 👋 

According to our [Supported Versions Policy](https://atlasgo.io/cli-reference#supported-version-policy), binaries for versions that were published more than 6 months ago will be removed from the CDN and Docker Hub. [v0.19.0](https://github.com/ariga/atlas/releases/tag/v0.19.0) was released 9 months ago, and it was deleted from our CDN.

**Comment by jufemaiz** [comment]
Ooof. One thing to cease active support, another to `rm -rf` the references altogether.

This will sadly have some significant consequences on our consideration of ent moving forward. A not insignificant change that seems to have only [recently been made](https://github.com/ariga/atlas/commit/7e4e09340681578747d787726e0d59e91925b8e1)?

**Comment by a8m** [comment]
Can you elaborates what blocks you from upgrading Atlas to its latest version?

**Comment by jufemaiz** [comment]
So we make use of [Hermit](https://github.com/cashapp/hermit) to maintain our releases of key dependencies to fixed (known + tested) versions. Particularly coming from a Golang environment, and being users of Ent, the idea that working versions are removed from availability so quickly (~8 months maximum support duration), and for something that is substantially a base layer (that is, should change relatively infrequently) tooling, is concerning. The first we knew of it was the answer here that was caused by breaking builds.

You use the word "block" – that's an absolute. I'm not certain this represents an absolute, but rather a mark against ongoing adoption of Atlas (including IMHO Ent's strong recommendation to adopt Atlas).

**Comment by rotemtam** [comment]
Hey @jufemaiz 

Thanks for raising your concern. I totally understand where you're coming from and apologize if we caused any inconvenience. 

Our main motivation for removing old versions from the CDN (which our company happily pays for, by the way) is that we fixed many bugs, upgraded libs with CVEs, and no longer support them.  We work very hard to keep Atlas BC, so honestly I don't see any reason for users to use versions that we know have issues. 

Being a tool that has access to critical resources like your production databases, I believe we should do what we can to ensure people are using recent versions to prevent issues, if possible. 

We have also invested a lot of time and effort for this policy to be transparent to users, including deprecation messages in the download script, at runtime and in the documentation.
